src/Symfony/Component/Console/Formatter/OutputFormatterStyleInterface.php

```diff
@@ -21,28 +23,36 @@ interface OutputFormatterStyleInterface
     /**
      * Sets style foreground color.
      *
-     * @param string $color The color name
+     * @param string|null $color The color name
+     *
+     * @throws InvalidArgumentException When the color name isn't defined
```

Comment:
Let's not add `@throws` annotations. We're not doing it everywhere and I don't want to go down this road.

Reply:
removed